controllers/iochaos/podiochaosmanager/podiochaosmanager.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package podiochaosmanager
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"golang.org/x/sync/errgroup"
+	v1 "k8s.io/api/core/v1"
+	k8sError "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/retry"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+)
+
+// PodIoManager will save all the related podnetworkchaos
```

Comment:
Just a remind about this: this PR may copied from podnetworkchaos implementation, and there are two many duplicate words in comments, which are stilling using `podnetworkchaos`, @YangKeao .

Reply:
I cannot wait to write a standalone pkg to unify the implementation of (podnetworkchaos|podiochaos)manager :rocket: , but it's still too busy to do it before 1.0 release :cry: .

Reply:
Em.. I means it's a copy-paste-issue, for example, `// PodIoManager will save all the related podnetworkchaos` should be `// PodIoManager will save all the related podiochaos`.